.github/CODEOWNERS

```diff
@@ -0,0 +1,16 @@
+# Require review from domain experts when the PR modified significant config files.
+/tests/integrations/config/test-custom.toml @tikv/tikv-configuration-reviewer
```

Comment:
This is only for testing, doesn't need to be reviewed.

Reply:
Fixed